lightning/src/ln/channel.rs

```diff
@@ -4295,8 +4302,18 @@ impl<SP: Deref> Channel<SP> where
 					order: self.context.resend_order.clone(),
 				})
 			}
+		} else if msg.next_local_commitment_number < next_counterparty_commitment_number {
```

Comment:
the case before this is `else if msg.next_local_commitment_number == next_counterparty_commitment_number - 1` which would qualify for this as well. Should this be `msg.next_local_commitment_number < next_counterparty_commitment_number - 1`?

Reply:
I just think it'd be more clear if this was `msg.next_local_commitment_number < next_counterparty_commitment_number - 1` since the statement before is `else if msg.next_local_commitment_number == next_counterparty_commitment_number - 1`, shouldn't have to rely on the `-1` in the previous statement to make this one correct.